source/docs/articles/wordpress/wordpress-best-practices.md

```diff
@@ -0,0 +1,25 @@
+---
+title: WordPress Best Practices
+description: A list of suggestions for developing WordPress sites on Pantheon.
+keywords: wordpress, best practices, redis, wp-cfm, behat
+---
+
+This article provides suggestions, tips, and best practices for developing and managing WordPress sites on the Pantheon platform.
+
+* Use [Redis](/docs/articles/sites/installing-redis-on-wordpress/). Redis is an open-source, networked, in-memory, key-value data store that can be used as a drop-in caching backend for your WordPress site. Pantheon makes it super simple and you'll be able to catch a lot of database queries in WordPress.
+
+* Use [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/) to aggregate JS/CSS on your side rather than relying on the server to do it for you. This helps speed up your workflow by minimizing redundant tasks.
```

Comment:
We should try standardize "on your side" to be "on local development/environment"